web/controllers/user_controller.ex

```diff
@@ -24,7 +24,9 @@ defmodule Org.UserController do
   end
 
   def show(conn, %{"id" => id}) do
-    user = Repo.get!(User, id)
+    user = User
+            |> Repo.get!(id)
```

Comment:
OK, I'll refactor this in another PR to remove the indents. Thanks for the feedback 👍